src/auth_api/endpoints/tokens.py

```diff
@@ -64,38 +74,57 @@ def get_internal_token(
 
 class InspectToken(Endpoint):
     """
-    TODO
+    Return the the internalToken.
```

Comment:
Change to `Return the InternalToken.` Note there are a couple of place in this file where a `InternalToken` is referenced as `internalToken`.